src/render/painter.js

```diff
@@ -160,6 +160,9 @@ class Painter {
     debugOverlayCanvas: HTMLCanvasElement;
     _terrain: ?Terrain;
     globeSharedBuffers: ?GlobeSharedBuffers;
+    globeCenterInViewSpace: [number, number, number];
+    globeRadius: number;
```

Comment:
Just out of curiosity: why did you end up adding `globeCenterInViewSpace` and `globeRadius` to `Painter` instead of `Transform`?

Reply:
No specific reason, I think these would fit better in the `Transform` so I'll move them there, thanks for pointing that up!